flow360/component/simulation/validation/validation_simulation_params.py

```diff
@@ -39,6 +40,46 @@ def _check_consistency_wall_function_and_surface_output(v):
     return v
 
 
+def _check_consistency_ddes_volume_output(v):
+    model_type = None
+    run_ddes = False
+
+    models = v.models
+
+    if not models:
+        return v
+
+    for model in models:
+        if isinstance(model, Fluid):
+            turbulence_model_solver = model.turbulence_model_solver
+            if not isinstance(turbulence_model_solver, NoneSolver) and turbulence_model_solver.DDES:
+                model_type = turbulence_model_solver.type_name
+                run_ddes = True
+                break
+
+    outputs = v.outputs
+
+    if not outputs:
+        return v
+
+    for output in outputs:
+        if isinstance(output, VolumeOutput) and output.output_fields is not None:
+            output_fields = output.output_fields.items
+            if "SpalartAllmaras_DDES" in output_fields and not (
+                model_type == "SpalartAllmaras" and run_ddes
```

Comment:
`run_ddes` and `model_type` may be referenced without definition when DDES is not activated or when user uses NoneSolver for turb solver.